defaults/initrd.scripts

```diff
@@ -2068,7 +2154,7 @@ openLUKS() {
 			fi
 
 			# At this point, {header,key}file or not, we're ready!
-			crypt_filter "${gpg_cmd}cryptsetup ${cryptsetup_options} luksOpen ${LUKS_DEVICE} ${LUKS_NAME}"
+			crypt_filter "${ASK_PASS}" "${gpg_cmd}cryptsetup ${cryptsetup_options} luksOpen ${LUKS_DEVICE} ${LUKS_NAME}"
```

Comment:
`ASK_PASS` should be at the second arg. Similar to `CRYPT_SILENT` flag for `{good,bad}_msg()`.